portfolio/src/main/webapp/script.js

```diff
@@ -27,35 +27,26 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
+//Generates a number between 1 and 200
+function randomNumber() {
+    return Math.floor((Math.random()*200)+1);
+}
+
 /*
- In order to access the secret page you have to solve a randomly generated equation.
- Upon the input of a correct answer a button will appear, linking the user to the page.
+ * In order to access the secret page you have to solve a randomly generated equation.
+ * Upon the input of a correct answer a button will appear, linking the user to the page.
  */
 function secretPage(){
   //Generates a number between 1 and 200 for both numbers.
-  this.a = Math.floor((Math.random()*200)+100);
-  this.b = Math.floor((Math.random()*200)+100);
+  var a = randomNumber();
```

Comment:
Instead of using `var`, which preceedes `const` and `let`, use either `const` or `let`. Since these values will be reassigned, `let` would be the right choice here.